File: 30-Days-Of-Code/swift/Day 3 : Intro to Conditional Statements.swift
import Foundation



guard let N = Int((readLine()?.trimmingCharacters(in: .whitespacesAndNewlines))!)
else { fatalError("Bad input") }

if N % 2 == 1 {
    print("Weird")
} else {
    N >= 6 && N <= 20 ? print("Weird") : print("Not Weird")
}

File: 30-Days-Of-Code/javascript/Day 3: Intro to Conditional Statements.js
function readLine() {
  let current = parseInt(inputString[currentLine++]);
  if (current % 2 === 1) {
      console.log('Weird');
  } else {
      current >= 6 && current <= 20 ? console.log('Weird') : console.log('Not Weird')
  }
}
